### PR TITLE
feat: FF-67: get_all_toggles method

### PR DESCRIPTION
**Pull request by FranLMSP**
#### 🤔 Why?

- We need a method to get the state of all the available toggles

#### 🛠 What I changed:

- Created a `get_all_toggles` method for the router

#### 🗃️ Jira Issues:

- [FF-67](https://ioetec.atlassian.net/browse/FF-67)

[FF-67]: https://ioetec.atlassian.net/browse/FF-67?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/router_unit_test.py [hunk 1]
```diff
@@ -102,3 +102,29 @@ def test__uses_the_toggle_context_when_it_is_provided(
...
+    def test__get_all_toggles(
+        self,
+        monkeypatch,
+        add_toggles: typing.Callable[[typing.Dict[str, bool]], None],
+        environment_name: str,
+    ):
+        monkeypatch.setenv("ENVIRONMENT", environment_name)
+        toggles = {
+            environment_name: {
+                "some_toggle": {"enabled": True},
```
Comment (Mikkel-14) [inline tests/router_unit_test.py:116]: One thing that got me thinking is what will happen with the non-static toggles, for example a pilot users toggle, that uses a toggle context

Comment (Mikkel-14) [inline tests/router_unit_test.py:116]: One thing that got me thinking is what will happen with the non-static toggles, for example a pilot users toggle, that uses a toggle context, can a test be added for that please?

**Review by Mikkel-14 — COMMENTED** [review]
Looks good, just left a small question 👍🏻

File: tests/router_unit_test.py (hunk 1 above)
Comment (FranLMSP) [reply tests/router_unit_test.py:116]: Sure! I can add those to the test

**Review by Mikkel-14 — APPROVED** [review]
🎏

**Pull request merged**